matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/call/model/MxCallImpl.kt

```diff
@@ -203,16 +203,19 @@ internal class MxCallImpl(
 
     override fun selectAnswer() {
         Timber.tag(loggerTag.value).v("select answer $callId")
-        if (isOutgoing) return
-        state = CallState.Answering
-        CallSelectAnswerContent(
-                callId = callId,
-                partyId = ourPartyId,
-                selectedPartyId = opponentPartyId?.getOrNull(),
-                version = MxCall.VOIP_PROTO_VERSION.toString()
-        )
-                .let { createEventAndLocalEcho(type = EventType.CALL_SELECT_ANSWER, roomId = roomId, content = it.toContent()) }
-                .also { eventSenderProcessor.postEvent(it) }
+        if (isOutgoing) {
```

Comment:
Would it be simpler to do
```
if (!isOutgoing) return
```
or whatever the Kotlin equivalent is?

Reply:
Yea you're right I could have kept the guard